api-docs/cloud/ref.yml

```
@@ -14635,9 +14810,14 @@ tags:
       indicates that InfluxDB accepted the request and request data is valid.
       Asynchronous operations, such as `write`, might not have completed yet. |
 
-      | `400`       | Bad request              | `Authorization` header is
-      missing or malformed or the API token does not have permission for the
-      operation. |
+      | `400`       | Bad request              | May indicate one of the
+      following: <li>Line protocol is malformed. The response body contains the
+      first malformed line in the data and indicates what was expected. For
+      partial writes, the number of points written and the number of points
+      rejected are also included. For more information, check the
+      `rejected_points` measurement in your `_monitoring`
+      bucket.</li><li>`Authorization` header is missing or malformed or the API
+      token does not have permission for the operation.</li> |
```

I know redoc will take care of it, but I think we should make the inline HTML valid.

```suggestion
      | `400`       | Bad request              | May indicate one of the
      following: <ul><li>Line protocol is malformed. The response body contains the
      first malformed line in the data and indicates what was expected. For
      partial writes, the number of points written and the number of points
      rejected are also included. For more information, check the
      `rejected_points` measurement in your `_monitoring`
      bucket.</li><li>`Authorization` header is missing or malformed or the API
      token does not have permission for the operation.</li></ul> |
```

---

> I know redoc will take care of it, but I think we should make the inline HTML valid.

I'll fix this in my next openapi PR